### PR TITLE
Add response headers policy input for CloudFront distributions default behaviour

### PR DESCRIPTION
**Pull request by itschrishudson**
## what
* Adds an input to pass a response headers policy ID into the CloudFront distribution in the default cache behaviour.

## why
* AWS have recently released the ability to generate response header policies to add response headers for CORS/Security and custom use cases instead of setting at the origin or using a Lambda.
* The current module does not allow you to pass in the ID of a policy in order to associate it with a the CloudFront distributions default behaviour.

## references
* https://aws.amazon.com/blogs/networking-and-content-delivery/amazon-cloudfront-introduces-response-headers-policies/
* https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudfront_distribution#response_headers_policy_id
* Part of solution to #192

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -175,6 +175,13 @@ variable "price_class" {
   description = "Price class for this distribution: `PriceClass_All`, `PriceClass_200`, `PriceClass_100`"
 }
 
+
```
Comment (mihaiplesa) [inline variables.tf:178]: extra newline not needed

Comment (itschrishudson) [reply variables.tf:178]: Fixed

File: variables.tf [hunk 2]
```diff
@@ -175,6 +175,12 @@ variable "price_class" {
   description = "Price class for this distribution: `PriceClass_All`, `PriceClass_200`, `PriceClass_100`"
 }
 
+variable "response_headers_policy_id" {
+  type        = string
+  description = "The identifier for a response headers policy - optional"
```
Comment (nitrocode) [inline variables.tf:180]:
```suggestion
  description = "The identifier for a response headers policy"
```

**Review by nitrocode — APPROVED** [review]

**Comment by nitrocode** [comment]
@itschrishudson thanks for the contribution. This has been released as https://github.com/cloudposse/terraform-aws-cloudfront-s3-cdn/releases/tag/0.76.0 🎉

**Pull request merged**